dspace-api/src/main/java/org/dspace/eperson/Group.java

```diff
@@ -203,7 +202,9 @@ public String getName()
 
     public void setName(Context context, String name) throws SQLException
     {
-        getGroupService().setMetadataSingleValue(context, this, MetadataSchema.DC_SCHEMA, "title", null, null, name);
+        if (!permanent)
+            getGroupService().setMetadataSingleValue(context, this,
+                    MetadataSchema.DC_SCHEMA, "title", null, null, name);
```

Comment:
I'd suggest we also log/throw an error here if anyone attempt to rename a permanent group. That way if the UI layer (or any caller) makes such an attempt, it will receive an error message in return, instead of no response.